sandboxed_containers/troubleshooting-sandboxed-containers.adoc

```diff
@@ -1,26 +1,30 @@
 :_content-type: ASSEMBLY
 [id="troubleshooting-sandboxed-containers"]
-= Collecting {sandboxed-containers-first} data for Red Hat Support
-include::_attributes/common-attributes.adoc[]
+= Collecting {sandboxed-containers-first} data
+include::modules/common-attributes.adoc[]
 :context: troubleshooting-sandboxed-containers
 
 toc::[]
 
 [role="_abstract"]
 
-When opening a support case, it is helpful to provide debugging
-information about your cluster to Red Hat Support.
+When troubleshooting {sandboxed-containers-first}, you can open a support case and provide debugging information using the `must-gather` tool.
 
-The `must-gather` tool enables you to collect diagnostic information about your
-{product-title} cluster, including virtual machines and other data related to
-{sandboxed-containers-first}.
+If you are a cluster administrator, you can also review logs on your own, enabling a more detailed level of logs.
 
-For prompt support, supply diagnostic information for both {product-title}
-and {sandboxed-containers-first}.
+include::modules/sandboxed-containers-collect-data-support.adoc[leveloffset=+1]
+include::modules/about-must-gather.adoc[leveloffset=+2]
 
-include::modules/about-must-gather.adoc[leveloffset=+1]
+To collect {sandboxed-containers-first} data with `must-gather`, you must specify the
+{sandboxed-containers-first} image:
+[source,terminal]
+----
+--image=registry.redhat.io/openshift-sandboxed-containers-tech-preview/osc-must-gather-rhel8:1.1.0
```

Comment:
Isn't the link changed in the 4.11 docs?

Reply:
Yes, the version updates are happening in a different PR.